core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -127,10 +127,41 @@ class LogManagerTest {
       // This should cause log1.close() to fail during LogManger shutdown sequence.
       FileUtils.deleteDirectory(logFile1)
 
-      logManagerForTest.get.shutdown()
+      logManagerForTest.get.shutdown(3)
 
-      assertFalse(Files.exists(new File(logDir1, LogLoader.CleanShutdownFile).toPath))
-      assertTrue(Files.exists(new File(logDir2, LogLoader.CleanShutdownFile).toPath))
+      assertFalse(Files.exists(new File(logDir1, CleanShutdownFileHandler.CleanShutdownFileName).toPath))
+      assertTrue(Files.exists(new File(logDir2, CleanShutdownFileHandler.CleanShutdownFileName).toPath))
+      assertEquals(-1L, logManagerForTest.get.readBrokerEpochFromCleanShutdownFiles())
+    } finally {
+      logManagerForTest.foreach(manager => manager.liveLogDirs.foreach(Utils.delete))
+    }
+  }
+
+  @Test
+  def testCleanShutdownFileWithBrokerEpoch(): Unit = {
+    // We create two directories logDir1 and logDir2 to help effectively test error handling
```

Comment:
The comment is bit mis-leading since there is no error handling in this test.

Reply:
Done